assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -175,19 +158,11 @@ def compute_explanations(model_wrapper, train_data, test_data, categorical_featu
     :return: explanation scores for the input data
     :rtype: list[float]
     """
-    # Create the RAI Insights object, split baseline data into train and test data
-    feature_metadata = FeatureMetadata(categorical_features=categorical_features, dropped_features=[])
-
-    rai_i: RAIInsights = RAIInsights(
-        model_wrapper, train_data, test_data, target_column, task_type, feature_metadata=feature_metadata
-    )
-    log_time_and_message("Created RAIInsights")
-    # Add the global explanations using batching to allow for larger input data sizes
-    rai_i.explainer.add()
-    evaluation_data = train_data.drop([target_column], axis=1)
+    evaluation_data = test_data.drop([target_column], axis=1)
```

Comment:
why test_data drop here? I think test data only has one column which is target column

Reply:
see line 139 in method get_train_test_data - we downsample the input dataset with line:

train_data, test_data = train_test_split(data, test_size=test_size, random_state=0)

we should have been using the test data but it looks like there was a mistake and train data was used instead.  Both train and test datasets in this case have feature columns and label column.